.github/workflows/build-and-upload-bottles.yml

```diff
@@ -0,0 +1,79 @@
+name: Build and Upload Homebrew Bottles
+
+on:
+  push:
+    branches:
+      - master
+      
+env:
```

Comment:
trusting you that this is the standard way to declare constants 👍

Reply:
I was following github documentation [here](https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow).